force-app/main/default/lwc/systemTools/systemTools.js

```diff
@@ -1,10 +1,82 @@
-import { LightningElement, wire, track } from "lwc";
-import { refreshApex } from "@salesforce/apex";
+import { LightningElement, wire, track, api } from "lwc";
+import { ShowToastEvent } from "lightning/platformShowToastEvent";
 
 import stgSystemToolsTitle from "@salesforce/label/c.stgSystemToolsTitle";
+import stgBtnRefreshHouseholdNames from "@salesforce/label/c.stgBtnRefreshHouseholdNames";
+import stgRefreshHHAcctNameTitle from "@salesforce/label/c.stgRefreshHHAcctNameTitle";
+import stgRefreshHHAcctNameDesc from "@salesforce/label/c.stgRefreshHHAcctNameDesc";
+import stgRefreshAdminNamesSuccessToast from "@salesforce/label/c.stgRefreshAdminNamesSuccessToast";
+import BatchJobRunningProblem from "@salesforce/label/c.BatchJobRunningProblem";
+import stgTellMeMoreLink from "@salesforce/label/c.stgTellMeMoreLink";
+
+import runRefreshHouseholdAccountNamingJob from "@salesforce/apex/AccountNamingBatchController.runRefreshHouseholdAccountNamingJob";
 
 export default class systemTools extends LightningElement {
     labelReference = {
         stgSystemToolsTitle: stgSystemToolsTitle,
+        stgBtnRefreshHouseholdNames: stgBtnRefreshHouseholdNames,
+        stgRefreshHHAcctNameTitle: stgRefreshHHAcctNameTitle,
+        stgRefreshHHAcctNameDesc: stgRefreshHHAcctNameDesc,
+        stgRefreshAdminNamesSuccessToast: stgRefreshAdminNamesSuccessToast,
+        BatchJobRunningProblem: BatchJobRunningProblem,
+        tellMeMoreLink: stgTellMeMoreLink,
     };
+
+    houseHoldNamesHyperLink =
+        '<a href="https://powerofus.force.com/s/article/EDA-Customize-Admin-and-HH-Acct-Names">' +
+        this.labelReference.tellMeMoreLink +
+        "</a>";
+
+    handleRefreshHHNamesBtnClick(event) {
+        const eventDetail = event.detail;
+        const batchJobToRun = "ACCT_HouseholdNameRefresh_BATCH";
+        this.dispatchSettingsBatchJobModalEvent(eventDetail, batchJobToRun);
+    }
+
+    dispatchSettingsBatchJobModalEvent(eventDetail, batchJobToRun) {
+        const settingsBatchJobDetail = {
+            eventDetail: eventDetail,
+            batchJobToRun: batchJobToRun,
+        };
+        this.dispatchEvent(
+            new CustomEvent("settingsbatchjobmodalrequest", {
+                detail: settingsBatchJobDetail,
+                bubbles: true,
+                composed: true,
+            })
+        );
+    }
+
+    @api modalSave(saveModel) {
+        switch (saveModel.batchJobToRun) {
+            case "ACCT_HouseholdNameRefresh_BATCH":
+                this.runHouseHoldNamingRefreshBatch();
+                break;
+        }
+    }
+
+    runHouseHoldNamingRefreshBatch() {
+        runRefreshHouseholdAccountNamingJob()
+            .then((result) => {
+                this.showToast("success", "Success", this.labelReference.stgRefreshAdminNamesSuccessToast);
```

Comment:
This is using the Admin Names success for the household names. We'll want to use the household names refresh toast for this. There are a number of other labels to align with on the spec.